python/setup.py.in

```diff
@@ -48,7 +48,7 @@ setup(name="opendht",
       ext_modules = cythonize(Extension(
           "opendht",
           ["@CURRENT_SOURCE_DIR@/opendht.pyx"],
-          include_dirs = ['@PROJECT_SOURCE_DIR@/include'],
+          include_dirs = ['@PROJECT_SOURCE_DIR@/include', '@PROJECT_SOURCE_DIR@/tools'],
```

Comment:
If log.h may be used by other components, let's move it to 'include'.
I'm not too comfortable with the python layer depending on something in 'tools'.